package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
are we planning to lint on the backend or just the frontend?

Reply:
Great question. Made me realize this linter doesn't work great for our needs. I migrated to a more gradle-style linter.